verl/trainer/config/_generated_ppo_megatron_trainer.yaml

```diff
@@ -54,6 +54,7 @@ actor_rollout_ref:
       override_mcore_model_config: {}
       use_mbridge: false
       vanilla_mbridge: true
+      thd_format: true
```

Comment:
We already have an option `use_remove_padding`, which is used by fsdp worker, but it seems like megatron doesn't respect it.